models/system/openai-whisper-large/model.yaml

```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/openai-whisper-large/27b5bb3/mlflow_model_folder
+  container_path: huggingface/openai-whisper-large/6fdb54a/mlflow_model_folder
```

Comment:
Looks like a lot of previous changes are present. You may need to take a latest pull from main